Gemfile

```diff
@@ -38,4 +39,4 @@ group :development do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
Why this EOF removal? maybe the editor remove it?

Reply:
Maybe I remove it

Reply:
Can you add it again please?